waspc/cli/src/Wasp/Cli/Command/CreateNewProject.hs

```diff
@@ -18,27 +18,41 @@ import Wasp.Cli.Command (Command, CommandError (..))
 import qualified Wasp.Cli.Command.Common as Command.Common
 import qualified Wasp.Cli.Common as Common
 import qualified Wasp.Data
-import Wasp.Util (indent)
+import Wasp.Util (indent, kebabToCamelCase)
 import qualified Wasp.Util.Terminal as Term
 
-newtype ProjectName = ProjectName {_projectName :: String}
+data ProjectInfo = ProjectInfo
+  { _projectName :: String,
+    _appName :: String
+  }
 
-createNewProject :: String -> Command ()
-createNewProject name
-  | isValidWaspIdentifier name = createNewProject' (ProjectName name)
+-- Returns either the ProjectInfo type that contains both the Project name
+-- and the App name (which might be the same), or an error describing why the name is invalid
```

Comment:
I would mention that the String it takes is a project name!
We can add that to comment, or we could even rename the function to `parseProjectNameIntoProjectInfo`.

Reply:
Done